src/utils/setup/remove-excluded-articles.js

```diff
@@ -0,0 +1,25 @@
+const removeExcludedArticles = (allArticles, excludedLearnPageArticles) => {
+    if (excludedLearnPageArticles && excludedLearnPageArticles.length) {
+        const filteredArticles = allArticles.filter(
+            article =>
+                !excludedLearnPageArticles.find(excludedArticle =>
+                    article.query_fields.slug.match(
+                        new RegExp(`^/?${excludedArticle}$`)
```

Comment:
I would have loved a small optimization here by making RegExp objects in the outer loop, but we don't know if these slugs will have slashes or not, so the regex would be more complicated.